### ISSUE TITLE
can't import mxnet in other folder

### ISSUE THREAD

**Issue by chechestars** [yajiedesign/mxnet#13]
I’ve installed anaconda3 andpy3,so I create an environment for py2 named”mxnet”.And I followed the steps ,but I find I can only import mxnet successfully when I activate “mxnet” environment and go to the folder “D:\mxnet\python”.I can not import mxnet in other folder.It really confused me when I use jupyter but I can’t import mxnet.I’m not sure is there any solution for that?

**Comment by yajiedesign** [comment]
you need install first.
run `python setup.py install ` in mxnet\python.

**Comment by chechestars** [comment]
Yes,I've followed the steps in https://stankirdey.com/2017/07/04/upgrading-mxnet-from-0-9-to-0-10-on-windows-10/#comment-133.But I've encontered such question.

**Comment by chechestars** [comment]
(mxnet) D:\mxnet\python>python setup.py install
running install
running bdist_egg
running egg_info
writing requirements to mxnet.egg-info\requires.txt
writing mxnet.egg-info\PKG-INFO
writing top-level names to mxnet.egg-info\top_level.txt
writing dependency_links to mxnet.egg-info\dependency_links.txt
reading manifest file 'mxnet.egg-info\SOURCES.txt'
writing manifest file 'mxnet.egg-info\SOURCES.txt'
installing library code to build\bdist.win-amd64\egg
running install_lib
warning: install_lib: 'build\lib' does not exist -- no Python modules to install

installing package data to build\bdist.win-amd64\egg
running install_data
creating build\bdist.win-amd64\egg
creating build\bdist.win-amd64\egg\mxnet
copying D:\mxnet\python\mxnet\..\..\build\libmxnet.dll -> build\bdist.win-amd64\egg\mxnet
creating build\bdist.win-amd64\egg\EGG-INFO
copying mxnet.egg-info\PKG-INFO -> build\bdist.win-amd64\egg\EGG-INFO
copying mxnet.egg-info\SOURCES.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying mxnet.egg-info\dependency_links.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying mxnet.egg-info\not-zip-safe -> build\bdist.win-amd64\egg\EGG-INFO
copying mxnet.egg-info\requires.txt -> build\bdist.win-amd64\egg\EGG-INFO
copying mxnet.egg-info\top_level.txt -> build\bdist.win-amd64\egg\EGG-INFO
creating 'dist\mxnet-0.10.1-py2.7.egg' and adding 'build\bdist.win-amd64\egg' to it
removing 'build\bdist.win-amd64\egg' (and everything under it)
Processing mxnet-0.10.1-py2.7.egg
removing 'c:\users\che\anaconda3\envs\mxnet\lib\site-packages\mxnet-0.10.1-py2.7.egg' (and everything under it)
creating c:\users\che\anaconda3\envs\mxnet\lib\site-packages\mxnet-0.10.1-py2.7.egg
Extracting mxnet-0.10.1-py2.7.egg to c:\users\che\anaconda3\envs\mxnet\lib\site-packages
mxnet 0.10.1 is already the active version in easy-install.pth

Installed c:\users\che\anaconda3\envs\mxnet\lib\site-packages\mxnet-0.10.1-py2.7.egg
Processing dependencies for mxnet==0.10.1
Searching for graphviz==0.8
Best match: graphviz 0.8
Processing graphviz-0.8-py2.7.egg
graphviz 0.8 is already the active version in easy-install.pth

Using c:\users\che\anaconda3\envs\mxnet\lib\site-packages\graphviz-0.8-py2.7.egg
Searching for requests==2.14.2
Best match: requests 2.14.2
Adding requests 2.14.2 to easy-install.pth file

Using c:\users\che\anaconda3\envs\mxnet\lib\site-packages
Searching for numpy==1.12.1
Best match: numpy 1.12.1
Adding numpy 1.12.1 to easy-install.pth file

Using c:\users\che\anaconda3\envs\mxnet\lib\site-packages
Finished processing dependencies for mxnet==0.10.1

**Comment by yajiedesign** [comment]
Any error messages when you import?

**Comment by chechestars** [comment]
(mxnet) D:\mxnet\python>python
Python 2.7.13 |Anaconda 4.4.0 (64-bit)| (default, May 11 2017, 13:17:26) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://anaconda.org
>>> import mxnet
>>> exit()

(mxnet) D:\mxnet\python>cd ..

(mxnet) D:\mxnet>python
Python 2.7.13 |Anaconda 4.4.0 (64-bit)| (default, May 11 2017, 13:17:26) [MSC v.1500 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
Anaconda is brought to you by Continuum Analytics.
Please check out: http://continuum.io/thanks and https://anaconda.org
>>> import mxnet
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: No module named mxnet
>>>

**Comment by yajiedesign** [comment]
queerish.
you can try use other env name.like py2.

**Comment by yajiedesign** [comment]
or try not use env .

**Comment by chechestars** [comment]
ok,well,when i change the env name,it still have the same problem.
